client/src/pages/create/CreatePage.tsx

```diff
@@ -106,7 +106,7 @@ export default function CreatePage() {
   };
 
   return appUser ? (
-    <PageLayout title="Create a game!">
+    <PageLayout title="Create a game!" loading={loadingAppUser}>
```

Comment:
thing to explore in future is is i can just useAuth in the PageLayout component so i dont have to do this individually for each page that needs to wait for appUser. Maybe just a flag that says PageLayout component do this for me cus its repetitive code